### PR TITLE
Comet documentation improvements

### LINKED ISSUE #13632: Improvements to CometLogger Documentation page
## 📚 Documentation

Hi all. I would like to make a few small improvements to the docs page for the CometLogger. These include

1. Adding instructions for logging media (audio, images, text)
2. Including a link to a Colab Notebook to try out the Logger
3. Adding a link to the Comet Docs for further  support/information

If all this is acceptable I can open a PR soon. 

Thank you! 
______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

- [**Metrics**](https://github.com/Lightning-AI/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/Lightning-AI/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/Lightning-AI/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/Lightning-AI/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

cc @borda @rohitgr7

### PR DESCRIPTION
**Pull request by DN6**
## What does this PR do?

This PR:

1. Updates the link to the Comet website
2. Adds snippets to the Comet Logger docstring to show
    a. How to log Lightingmodule Hyperparameters via `self.save_hyperparameters()`
    b. How to log other experiment hyperparameters through the logger
    c. How to access the Comet Experiment object and use the underlying methods to log additional data (image, audio, 
        text, data assets, models)
3. Adds a link to a Demo Comet + Lightning Colab Notebook
4. Adds a link to the Pytorch Lightning Integration page in the Comet Documentation  

The following links the related issue to the PR:
https://github.com/Lightning-AI/lightning/issues/13632 

Fixes #13632

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by DN6** [comment]
Hi all. There are some failing CI tests related to testing on different accelerators (GPU, TPU, etc) but they don't seem related to my PR.  Please let me know if there's anything I need to address on my side.

**Review by awaelchli — APPROVED** [review]
Great! 
We've done a similar thing for other loggers too, but ideally IMO we would separate the user guide from the api docs. 
Maybe in the future we can restructure our user docs such that each logger integration can get their own page to showcase their features, while keeping the api docs clean and simple. 
cc @Felonious-Spellfire

**Review by carmocca — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]
